server/bin/pbench-check-tb-age.py

```diff
@@ -37,7 +38,7 @@ def main(options):
         )
         return 2
     tb_path = os.path.realpath(options.tb_path)
```

Comment:
Can you consider making `tb_path` be a `Path()` object, perhaps via `Path(os.path.realpath(options.tb_path))`?  It seems like you'll find that one change will make it more natural to use `pathlib` here.

Reply:
ok

Reply:
Creating it a path object was creating problem. I will see